packages/data-driver-postgres/src/utils/escape-identifier.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Temporarily util until the issue following issue is fixed {@link https://github.com/brianc/node-postgres/issues/3001}
+ * Because we're going to use the util function from node-postgres, there are no tests here for this function.
+ * @param identifier - like a table or column name
+ * @returns escaped identifier
+ */
+export function escapeIdentifier(identifier: string): string {
+	return `"${identifier.replaceAll('"', '""')}"`;
+}
```

Comment:
Since this is such a tiny one-liner, we will probably need similar functions for all vendors and the mechanism to escape identifiers in postgress is well-defined, simple and easy to test, I think we should just keep this util in our own codebase.

That being said, the [postgres docs](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#id-1.5.3.5.8.9.1) mention that quoted identifiers must not contain zero bytes. AFAIK postgres will error if it encounters such a zero byte. Do we still want to test for this in here and remove them or error ourselves?

Reply:
Would be fine by me to keep the function in our codebase. It would feel a little redundant though if we leave the function as it is.

To send a zero code character within an identifier seems to be very much an edge case 🤔 If I get it right, one would need to create an identifier string from a byte code to have that character included. For me it would be fine to leave that up to the database to handle. But I now added the error message from `node-postgres` into the error message the driver throws, so the user would know that this is not allowed - if it returned an appropriate error. This might be something to include in the integration tests.

Reply:
I'll remove the comment and add tests for that function, to keep it in our codebase

Reply:
I integrated the escape function now completely @nickrum